webapi/src/test/scala/org/knora/webapi/e2e/admin/PermissionsADME2ESpec.scala

```diff
@@ -215,10 +215,10 @@ class PermissionsADME2ESpec extends E2ESpec(PermissionsADME2ESpec.config) with T
             text = responseToString(response)
           )
         )
-
+        val customAPIri = "http://rdfh.ch/permissions/0001/AP-with-customIri"
```

Comment:
Shouldn't this also end with a Base64 UUID?

Reply:
yeah, this is not really making any request. It is only used for writing request and response to generated client test data. I will fix it though.

Reply:
ok